bin/shell

```diff
@@ -39,17 +39,14 @@ do_bindmount /run /run
 do_bindmount /proc /proc
 do_bindmount /sys /sys
 do_bindmount /dev /dev
-do_bindmount /dev/pts /dev/pts
```

Comment:
Aha, I see now that this is deliberate. The PR description is a bit misleading ;) But that is fine, its a bit terrible that we ned this hack but its easy enough to fix once we have a better solution.